src/VertexArray.cpp

```diff
@@ -121,6 +138,7 @@ VertexArray::SetVertex(const int aIndex, const Vector& aPoint) {
   if (m.vertices.size() < (aIndex + 1)) {
     m.vertices.resize(aIndex + 1);
   }
+  m.UpdateBoundingBox(aPoint);
```

Comment:
If the old vertex was already the max or min and the edited vertex is inside the bounding box would not be updated correctly

Reply:
I just removed it for now and will wait until we do bounding volume for real.